### ISSUE TITLE
Names are inconsistent

### ISSUE THREAD

**Issue by domenic** [w3c/longtasks#53]
same-origin vs. cross-origin-unreachable: why do only one of these have -unreachable?

multiple-contexts: this is the only one that uses the "-contexts" suffix. Can it just be "multiple"?

I can understand if the answer is "let's not change this because we shipped already". In that case we may want to add a note explaining that the inconsistency is historical, not intentional.

**Comment by npm1** [comment]
For "multiple-contexts" I think the idea is that from the other keywords it can be deduced that we're talking about a context, whereas "multiple" is a bit less clear. For "cross-origin-unreachable", not sure why it differs from "same-origin". But yea I think we might want to leave them as they are now to avoid breaking current users. We could have a small note if you think it's necessary.

**Comment by tdresser** [comment]
Domenic, do you think we should modify these, and risk breakage?
Or add a note in the spec?

Or should we just leave it as is?

**Comment by domenic** [comment]
I don't feel qualified to make that judgement, as I don't know the current usage, or how centralized and easy to update it is. I think the current names are pretty bad, but in the end they're just names; we shipped webkitMatchesSelector and it was fine.

One thing that might tip the balance is if other implementers were going to implement, but noticed the names were weird, and wanted to take the transition from single-implementer -> multi-implementer as an opportunity to fix them. I'm not sure if any of them feel that way.

**Comment by tdresser** [comment]
@toddreifsteck @yoavweiss 

Any thoughts?

**Comment by toddreifsteck** [comment]
@rniwa You are likely best positioned to have an opinion.

**Comment by rniwa** [comment]
Yeah, it seems redundant to say "unreachable". "same-origin" and "cross-origin" seems fine.

**Comment by yoavweiss** [comment]
[Usage](https://chromestatus.com/metrics/feature/timeline/popularity/1615) is high, but we currently don't have specific counters for how many `cross-origin-unreachable` names we're seeing. Might be worthwhile to add ones.

It may also be interesting to hear from users if the name is used at all, and what would happen if we changed it. 
@nicjansma - do you have opinions on that, with your RUM provider hat on?
